scripts/install-kusion.sh

```diff
@@ -22,7 +22,7 @@ for arg in "$@"; do
     fi
```

Comment:
The tr command to remove newlines is redundant since base64 encoding already produces a single line output when using the -w 0 option.